wiki/en/Directories.md

```diff
@@ -1,22 +1,22 @@
 ---
 layout: wiki
-title: "Custom Directories"
+title: "Directories"
```

Comment:
This needs a redirect page, since we change the permalink.

Reply:
Does that mean changing this:

https://github.com/gilgongo/jamweb-gilgongo/blob/next-release/wiki/en-no-translate/Directory-Servers.md

To contain:
```
---
layout: redirect
redirect: "/wiki/Directories"
lang: "en"
permalink: "/wiki/Directory-Servers"
category: "hidden"
---
```

Reply:
Yes.